data/text/common_2.asm

```diff
@@ -1145,39 +1150,39 @@ _PlayersPCNoRoomDepositText::
 	line "store items."
 	prompt
 
-UnknownText_0x1c144d::
+_PokecenterPCTurnOnText::
 	text "<PLAYER> turned on"
 	line "the PC."
 	prompt
 
-UnknownText_0x1c1462::
+_PokecenterPCWhoseText::
 	text "Access whose PC?"
 	done
 
-UnknownText_0x1c1474::
+_PokecenterPCBillText::
 	text "BILL's PC"
 	line "accessed."
 
 	para "#MON Storage"
 	line "System opened."
 	prompt
 
-UnknownText_0x1c14a4::
+_PokecenterPCPlayersText::
```

Comment:
_PokecenterPCOwnText

Reply:
I'll have to disagree on this one. That really isn't much clearer, and strays from the common theme of the rest of the labels. How about either `PokecenterPCPlayerText` or `PokecenterPCPlayersPCText`?

Reply:
PokecenterPlayersPCText is fine, and then PokecenterBillsPCText and PokecenterOaksPCText.